config/core/build-configs.yaml

```diff
@@ -177,6 +177,35 @@ fragments:
       - 'CONFIG_VIDEO_VIVID=y'
       - 'CONFIG_VIDEO_VIVID_MAX_DEVS=64'
 
+  x86_4.19.y_cip:
```

Comment:
Would it be possible to have a config fragment that didn't depend on the kernel revision?  The `merge_config.sh` script will ignore config names that don't exist, so if you combine fragments for newer kernels with older ones it might work.

Alternatively, you could have one common fragment and then smaller ones specific to some particular kernel versions.